docs/DeveloperGuide.md

```diff
@@ -401,23 +421,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * Steps 2a1-2a2 are repeated until all details entered are correct.
 * Use case resumes from step 3.
 
-**Use case: UC05 - Find student entries**
+**Use case: UC07 - Settle outstanding fees for student**
 
 **MSS**
 
-1. User enters command to find students.
-1. System displays list with students with matching details.
+1. User requests to <ins>list students(UC02)</ins>.
+1. User enters command to settle outstanding fees for the specified student. 
+1. System updates the total tuition fee paid and total tuition fee owed by the student.
+1. System displays success message.
 
    Use case ends.
 
 **Extensions**
```

Comment:
Could there be this possible extension where list returns nothing or student to settle payment is not in the list?

Reply:
Thank you for the suggestion. I have changed the usecase to use find and added in the extension where specified student is not found.